Steamworks/src/org/usfirst/frc/team4786/robot/Robot.java

```diff
@@ -32,9 +33,8 @@
 	@Override
 	public void robotInit() {
 		oi = new OI();
-//		chooser.addDefault("Default Auto", new ExampleCommand());
+		chooser.addObject("Test PID", new DriveToPosition(1));
```

Comment:
Scratch my earlier comment. The entire sendableChooser mechanism is redundant, as it is not used anywhere else. The chooser was bypassed for running the autonomous command. Again, I think that this is fine for now, but I will write it into an issue if this passes review.